From 769d4a6f6209fd6eead439aa6b50af144f8110fe Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Sat, 27 Feb 2021 01:07:07 +0000
Subject: [PATCH 4/4] Accept args for test runner

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 8f5e1e3..a6d5329 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -423,7 +423,7 @@ distclean: clean
 .PHONY: distclean
 
 test: $(REDIS_SERVER_NAME) $(REDIS_CHECK_AOF_NAME) $(REDIS_CLI_NAME) $(REDIS_BENCHMARK_NAME)
-	@(cd ..; ./runtest)
+	@(cd ..; ./runtest $(TEST_RUNNER_ARGS))
 
 test-modules: $(REDIS_SERVER_NAME)
 	@(cd ..; ./runtest-moduleapi)
-- 
2.36.0

